config/hiveadmission/deployment.yaml

```diff
@@ -36,7 +36,6 @@ spec:
         - "--audit-log-path=-"
         - "--tls-cert-file=/var/serving-cert/tls.crt"
         - "--tls-private-key-file=/var/serving-cert/tls.key"
-        - "--v=2"
```

Comment:
why this change?

Reply:
upstream removed all the logging options from the CLI we inherit. I didn't bother to track down when or why.